index.js

```diff
@@ -244,6 +244,19 @@ class Mapeo extends events.EventEmitter {
     })
   }
 
+  getDatabaseStatus () {
+    var res = []
+    var feeds = this.osm.core._logs.feeds()
```

Comment:
@noffle could this be in `kappa-osm`?

Reply:
It could. Honestly, it *should*. I've been putting this off until after we get Frando's big kappa rewrite in before adding it. Does that work for you? We can go through and replace all of these then. I really believed that nobody would ever need to access the underlyling multifeed, lol.

Reply:
sounds good to me!

Reply:
haha right, it's funny how things shake out sometimes